jdk_net and jdk_nio fails on linux with java8

**No issues in JBS:**
java/net/InetAddress/BadDottedIPAddress.java failed with 'UnknownHostException was not thrown for: 999.999.999.999'

```
Stacktrace

Execution failed: `main' threw exception: java.lang.Exception: UnknownHostException was not thrown for: 999.999.999.999    

Standard Output

    

Standard Error

java.lang.Exception: UnknownHostException was not thrown for: 999.999.999.999
	at BadDottedIPAddress.main(BadDottedIPAddress.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.sun.javatest.regtest.agent.MainActionHelper$AgentVMRunnable.run(MainActionHelper.java:298)
	at java.lang.Thread.run(Thread.java:748)

```
https://ci.adoptopenjdk.net/view/Test_openjdk/job/openjdk8_hs_openjdktest_x86-64_linux/306/testReport/junit/java_net_InetAddress_BadDottedIPAddress/java/BadDottedIPAddress/


**With issue opened in JBS:**
java/net/InetAddress/CachedUnknownHostName.java failed with 'java.lang.Exception: Missing java.net.UnknownHostException    '

https://ci.adoptopenjdk.net/view/Test_openjdk/job/openjdk8_hs_openjdktest_x86-64_linux/306/testReport/junit/java_net_InetAddress_CachedUnknownHostName/java/CachedUnknownHostName/

```
Standard Error

Missing java.net.UnknownHostException for host bogusHostName
java.lang.Exception: Missing java.net.UnknownHostException
	at CachedUnknownHostName.main(CachedUnknownHostName.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.sun.javatest.regtest.agent.MainActionHelper$AgentVMRunnable.run(MainActionHelper.java:298)
	at java.lang.Thread.run(Thread.java:748)

```
https://bugs.openjdk.java.net/browse/JDK-8056988 opened for window, should update with linux too.

java/net/ipv6tests/B6521014.java
java/net/MulticastSocket/SetGetNetworkInterfaceTest.java on linuxgot 'Cannot assign requested address 'Test failed: cannot create socket'

```
Standard Error

java.lang.RuntimeException: Test failed: cannot create socket.
	at B6521014.main(B6521014.java:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.sun.javatest.regtest.agent.MainActionHelper$AgentVMRunnable.run(MainActionHelper.java:298)
	at java.lang.Thread.run(Thread.java:813)
Caused by: java.net.BindException: Cannot assign requested address (Bind failed)
	at java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:387)
	at java.net.Socket.bind(Socket.java:644)
	at B6521014.test2(B6521014.java:103)
	at B6521014.main(B6521014.java:121)
	... 6 more
```

JBS: https://bugs.openjdk.java.net/browse/JDK-8006560

java/net/Inet6Address/B6206527.java 'Cannot assign requested address (Bind failed) '

```
java.net.BindException: Cannot assign requested address (Bind failed)
	at java.net.PlainSocketImpl.socketBind(Native Method)
	at java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:387)
	at java.net.ServerSocket.bind(ServerSocket.java:375)
	at java.net.ServerSocket.bind(ServerSocket.java:329)
	at B6206527.main(B6206527.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.sun.javatest.regtest.agent.MainActionHelper$AgentVMRunnable.run(MainActionHelper.java:298)
	at java.lang.Thread.run(Thread.java:813)

```

https://ci.adoptopenjdk.net/view/Test_openjdk/job/openjdk8_j9_openjdktest_x86-64_linux/433/testReport/junit/java_net_Inet6Address_B6206527/java/B6206527/
JBS: https://bugs.openjdk.java.net/browse/JDK-6670327

--- a/openjdk/excludes/ProblemList_openjdk21.txt
+++ b/openjdk/excludes/ProblemList_openjdk21.txt
@@ -153,7 +153,6 @@ java/net/Socket/ProxyCons.java https://github.com/adoptium/aqa-tests/issues/3088
 java/net/Socket/ReadTimeout.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le
 java/net/Socket/SetSoLinger.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le
 java/net/Socket/SoTimeout.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le
-java/net/Socket/asyncClose/AsyncClose.java https://github.com/adoptium/aqa-tests/issues/827 aix-ppc64, linux-ppc64le
 java/net/Socket/asyncClose/BrokenPipe.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le
 java/net/Socket/setReuseAddress/Basic.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le
 java/net/Socket/setReuseAddress/Restart.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le

--- a/openjdk/excludes/ProblemList_openjdk23.txt
+++ b/openjdk/excludes/ProblemList_openjdk23.txt
@@ -163,7 +163,6 @@ java/net/Socket/ProxyCons.java https://github.com/adoptium/aqa-tests/issues/3088
 java/net/Socket/ReadTimeout.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le
 java/net/Socket/SetSoLinger.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le
 java/net/Socket/SoTimeout.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le
-java/net/Socket/asyncClose/AsyncClose.java https://github.com/adoptium/aqa-tests/issues/827 aix-ppc64, linux-ppc64le
 java/net/Socket/asyncClose/BrokenPipe.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le
 java/net/Socket/setReuseAddress/Basic.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le
 java/net/Socket/setReuseAddress/Restart.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le

--- a/openjdk/excludes/ProblemList_openjdk24.txt
+++ b/openjdk/excludes/ProblemList_openjdk24.txt
@@ -163,7 +163,6 @@ java/net/Socket/ProxyCons.java https://github.com/adoptium/aqa-tests/issues/3088
 java/net/Socket/ReadTimeout.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le
 java/net/Socket/SetSoLinger.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le
 java/net/Socket/SoTimeout.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le
-java/net/Socket/asyncClose/AsyncClose.java https://github.com/adoptium/aqa-tests/issues/827 aix-ppc64, linux-ppc64le
 java/net/Socket/asyncClose/BrokenPipe.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le
 java/net/Socket/setReuseAddress/Basic.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le
 java/net/Socket/setReuseAddress/Restart.java https://github.com/adoptium/aqa-tests/issues/3088 linux-ppc64le

--- a/openjdk/excludes/ProblemList_openjdk8.txt
+++ b/openjdk/excludes/ProblemList_openjdk8.txt
@@ -173,12 +173,8 @@ java/math/BigInteger/LargeValueExceptions.java https://github.com/adoptium/aqa-t
 # jdk_net
 
 java/net/Inet4Address/PingThis.java https://github.com/adoptium/infrastructure/issues/1127 aix-all
-java/net/InetAddress/BadDottedIPAddress.java https://github.com/adoptium/aqa-tests/issues/827 linux-all
 java/net/Inet6Address/B6206527.java https://github.com/adoptium/infrastructure/issues/1085 macosx-all
-java/net/Inet6Address/B6558853.java https://github.com/adoptium/aqa-tests/issues/827 macosx-all
-java/net/InetAddress/CachedUnknownHostName.java https://github.com/adoptium/aqa-tests/issues/827 linux-all
 java/net/InetAddress/CheckJNI.java https://github.com/adoptium/infrastructure/issues/1085 macosx-all
-java/net/InetAddress/IPv4Formats.java https://github.com/adoptium/aqa-tests/issues/827 linux-all
 java/net/MulticastSocket/Promiscuous.java https://github.com/adoptium/infrastructure/issues/699 linux-s390x
 java/net/MulticastSocket/SetGetNetworkInterfaceTest.java https://github.com/adoptium/aqa-tests/issues/1011 aix-all
 # java/net/MulticastSocket/SetLoopbackMode.java on aix issue https://github.com/adoptium/aqa-tests/issues/1011
@@ -186,12 +182,9 @@ java/net/MulticastSocket/SetLoopbackMode.java https://github.com/adoptium/infras
 # java/net/MulticastSocket/Test.java https://github.com/adoptium/aqa-tests/issues/1011 aix-all
 java/net/MulticastSocket/Test.java https://github.com/adoptium/infrastructure/issues/699 linux-s390x,aix-all,macosx-all
 java/net/Socket/LinkLocal.java https://github.com/adoptium/infrastructure/issues/1085 macosx-all
-java/net/SocketPermission/Wildcard.java https://github.com/adoptium/aqa-tests/issues/827 linux-all
-java/net/Socks/SocksV4Test.java https://github.com/adoptium/aqa-tests/issues/827 linux-all
 java/net/ipv6tests/B6521014.java https://github.com/adoptium/infrastructure/issues/1085 macosx-all
 java/net/ipv6tests/UdpTest.java https://bugs.openjdk.java.net/browse/JDK-8198266 generic-all
 java/net/ipv6tests/TcpTest.java https://github.com/adoptium/infrastructure/issues/1085 macosx-all
-com/sun/net/httpserver/Test1.java https://github.com/adoptium/aqa-tests/issues/827 linux-ppc64le
 com/sun/net/httpserver/Test9a.java https://github.com/adoptium/aqa-tests/issues/4679 linux-arm
 com/sun/net/httpserver/Test12.java https://github.com/adoptium/aqa-tests/issues/4679 linux-arm
 sun/net/www/protocol/https/HttpsURLConnection/ImpactOnSNI.java https://bugs.openjdk.java.net/browse/JDK-8155049 solaris-x64
@@ -215,7 +208,6 @@ java/nio/channels/DatagramChannel/Promiscuous.java https://github.com/adoptium/i
 java/nio/channels/SocketChannel/Connect.java https://github.com/adoptium/aqa-tests/issues/156 macosx-all
 java/nio/channels/ServerSocketChannel/AdaptServerSocket.java https://github.com/adoptium/aqa-tests/issues/821 windows-all
 java/nio/channels/ServerSocketChannel/Basic.java https://github.com/adoptium/aqa-tests/issues/821 windows-all
-java/nio/channels/DatagramChannel/SendToUnresolved.java https://github.com/adoptium/aqa-tests/issues/827 linux-all
 java/nio/channels/Selector/RacyDeregister.java https://bugs.openjdk.java.net/browse/JDK-8161083 aix-all
 java/nio/channels/AsyncCloseAndInterrupt.java https://github.com/adoptium/aqa-tests/issues/593 aix-all
 sun/nio/ch/TestMaxCachedBufferSize.java https://github.com/adoptium/aqa-tests/issues/1051 linux-s390x

